tests/test_metrics_alerts.py

```diff
@@ -396,20 +396,20 @@ def run_queries_with_priority():
         kubectl.kubectl(f"exec {priority_pod} -- bash -c \"{cmd}\"", timeout=120)
         clickhouse.clickhouse_query(
             chi["metadata"]["name"],
-            "SELECT event_time, CurrentMetric_QueryPreempted FROM system.metric_log WHERE CurrentMetric_QueryPreempted > 0",
+            "SELECT event_time, CurrentMetric_ClickHouseQueryPreempted FROM system.metric_log WHERE CurrentMetric_ClickHouseQueryPreempted > 0",
```

Comment:
please rollback this rename to `CurrentMetric_QueryPreemted`
cause it's a system.metric_log field name and not AlertName